src/DssSpell.t.sol

```diff
@@ -28,7 +68,112 @@ contract DssSpellTest is DssSpellTestBase {
+    }
+
+
+    uint256 constant MAR_01_2022            = 1646092800; // 2022-03-01
+    uint256 constant APR_01_2022            = 1648771200; // 2022-04-01
+    function testDaiStreams() public {
+        uint256 streams = vestDai.ids();
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        assertEq(vestDai.cap(), 1 * MILLION * WAD / 30 days);
+        assertEq(vestDai.ids(), streams + 3);
+
+
+        vestDai.valid(30); // check for valid contract
+        checkDaiVest({
+            _index:      30,                          // id
+            _wallet:     wallets.addr("RISK_WALLET"), // usr
+            _start:      MAR_01_2022,                 // bgn
+            _cliff:      MAR_01_2022,                 // clf
+            _end:        MAR_01_2022 + 364 days,      // fin
+            _manager:    address(0),                  // mgr
+            _restricted: 1,                           // res
+            _reward:     2_760_000 * WAD,             // tot
+            _claimed:    0                            // rxd
+        });
+        vestDai.valid(31); // check for valid contract
+        checkDaiVest({
+            _index:      31,                          // id
+            _wallet:     wallets.addr("CES_WALLET"),  // usr
+            _start:      APR_01_2022,                 // bgn
+            _cliff:      APR_01_2022,                 // clf
+            _end:        APR_01_2022 + 364 days,      // fin
+            _manager:    address(0),                  // mgr
+            _restricted: 1,                           // res
+            _reward:     2_780_562 * WAD,             // tot
+            _claimed:    0                            // rxd
+        });
+        vestDai.valid(32); // check for valid contract
+        checkDaiVest({
+            _index:      32,                          // id
+            _wallet:     wallets.addr("IS_WALLET"),   // usr
+            _start:      MAR_01_2022,                 // bgn
+            _cliff:      MAR_01_2022,                 // clf
+            _end:        MAR_01_2022 + 275 days,      // fin
+            _manager:    address(0),                  // mgr
+            _restricted: 1,                           // res
+            _reward:     207_000 * WAD,               // tot
+            _claimed:    0                            // rxd
+        });
+    }
+
+    // https://github.com/makerdao/spells-mainnet/tree/master/archive/2021-09-03-DssSpell
+    address immutable RISK_OLD_WALLET = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
+    uint256 constant SEP_01_2021    = 1630454400;
+    uint256 constant SEP_01_2022    = 1661990400;
+    uint256 constant VEST_AMOUNT    = 2_184_000 * WAD;
+    uint256 constant CLAIMED_AMOUNT =   910_000 * WAD;
+
+    function testYankedDaiStreams() public {
+
+        vestDai.valid(8); // check for valid contract
```

Comment:
This is not testing anything. Should be `assertTrue(vestDai.valid(8));`.

Reply:
Done in 08dae9e.